File: Kol_2/2025_Kolokvijum_2_termin_2/Client/models/Stovariste.js
export class Stovariste {
  constructor(id, naziv, adresa, brojTelefona, materijaliUMagacinu = []) {
    this.id = id;
    this.naziv = naziv;
    this.adresa = adresa;
    this.brojTelefona = brojTelefona;
    this.materijaliUMagacinu = materijaliUMagacinu;
  }
}

File: Kol_2/2025_Kolokvijum_2_termin_2/StovaristeServer/Models/Stovariste.cs
namespace StovaristeServer.Models;

public class Stovariste
{
    public int ID { get; set; }
    public required string Naziv { get; set; }
    public required string Adresa { get; set; }
    public required string BrojTelefona { get; set; }
    public List<Magacin>? MaterijaliUMagacinu { get; set; }
}
